### ISSUE TITLE
Dropdown menu focus

### ISSUE THREAD

**Issue by basz** [kaliber5/ember-bootstrap#972]
Hi,

after and upgrade which updated bootstrap to 4.4.1 I seem to have focus around dropdown-menus. I've noticed the demo dropdown also does this. However, that does not seem right to me... Looking at getbootstrap's examples. focus is retained around the popup button.

```
➜  lab-plhw git:(master) ember bootstrap:info
Npm packages:
ember-bootstrap: ^3.1.0 -> 3.1.1
ember-cli: ~3.14.0 -> 3.14.0
bootstrap: ^4.3.1 -> 4.4.1
bootstrap-sass: n/a
ember-cli-sass: ^9.0.0 -> 9.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapCSS: false
importBootstrapTheme: false
importBootstrapFont: false
whitelist: undefined
```

**Comment by simonihmig** [comment]
Yup, that doesn't look right!

I think this was introduced in #942. We do have to handle focus explicitly when the dropdown is wormholed, as the dropdown menu's DOM is in a totally different place than the trigger/button. So won't ever receive focus the normal way. 

Tbh that optional wormhole feature causes quite some trouble here, for keyboard control and focus trapping... 🙁

**Comment by simonihmig** [comment]
This has been fixed in v3.1.3!
